### PR TITLE
Replaced ActionableMessage Instances in edit-gas

### PR DESCRIPTION
**Pull request by pritam1813**
## Explanation
- This PR is part of #19528 
- This PR replaces instances of `ActionableMessage` with `BannerAlert` in `ui/components/app/edit-gas-display/edit-gas-display.component.js`

### Before

Before
Before2

### After
Screenshot 2023-07-28 at 4 11 20 PM
Screenshot 2023-07-28 at 4 13 26 PM

## Manual Testing Steps

1. Build the project with `yarn dist`
2. Install the build to browser
3. Go to [Metamask Dapp Test Page](https://metamask.github.io/test-dapp/) and connect the wallet.
4. Click **SEND LEGACY TRANSACTION**
5. Edit the **Estimated gas fee**
6. Right click and Inspect the warning element.

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Comment by pritam1813** [comment]
Hi @georgewrmarshall , please review this.

**Review by georgewrmarshall — CHANGES_REQUESTED** [review]
Hey @pritam1813, this is looking good but I believe the after screenshots in the before/after section is not a good representation of the changes. It looks like the same screenshots. Could you please update the After section with the correct screenshots and also add some manual testing steps. Thanks

**Comment by pritam1813** [comment]
Hi @georgewrmarshall , Sorry , I thought that there isn't much visual difference in `BannerAlert` with `Severity` warning and `ActionableMessage` with type `Danger`, but,  actually it was due to the `className="actionable-message--warning"` which was interfering with the `BannerAlert` styling. I have changed the `className` to `"banner-alert--warning"`, I hope that's OK. 

 I have also changed the `ErrorMessage` element to `BannerAlert` `Severity='danger'`, which was displaying the insufficient balance error message.

File: ui/components/app/edit-gas-display/edit-gas-display.component.js [hunk 1]
```diff
@@ -62,38 +61,30 @@ export default function EditGasDisplay({
 
   const showTopError = balanceError && txParamsHaveBeenCustomized;
 
-  let errorKey;
-  if (balanceError) {
-    errorKey = 'insufficientFunds';
-  }
```
Comment (georgewrmarshall) [inline ui/components/app/edit-gas-display/edit-gas-display.component.js:68]: Moved to line `70`

File: ui/components/app/edit-gas-display/edit-gas-display.component.js [hunk 2]
```diff
@@ -62,38 +61,30 @@ export default function EditGasDisplay({
...
-
   return (
     <div className="edit-gas-display">
       <div className="edit-gas-display__content">
         {showTopError && (
-          <div className="edit-gas-display__warning">
```
Comment (georgewrmarshall) [inline ui/components/app/edit-gas-display/edit-gas-display.component.js:74]: Can reduce nesting and utilize style utility props for margin styles

File: ui/components/app/edit-gas-display/index.scss [hunk 3]
```diff
@@ -1,13 +1,4 @@
 .edit-gas-display {
-  & .actionable-message--warning,
-  & .actionable-message--error {
-    margin-top: 0;
-
-    & .actionable-message__message {
-      text-align: start;
-    }
-  }
```
Comment (georgewrmarshall) [inline ui/components/app/edit-gas-display/index.scss:9]: No longer using the components that use these class names

File: ui/components/app/edit-gas-display/index.scss [hunk 4]
```diff
@@ -21,10 +12,6 @@
     }
   }
 
-  &__dapp-acknowledgement-warning {
-    margin-bottom: 20px;
-  }
```
Comment (georgewrmarshall) [inline ui/components/app/edit-gas-display/index.scss:26]: No longer using this classname
Screenshot 2023-07-28 at 4 08 41 PM

File: ui/components/app/edit-gas-display/index.scss [hunk 5]
```diff
@@ -39,35 +26,10 @@
     margin: 20px auto;
   }
 
-  &__advanced-button {
```
Comment (georgewrmarshall) [inline ui/components/app/edit-gas-display/index.scss:42]: No classname exists for these styles
Screenshot 2023-07-28 at 3 54 46 PM

File: ui/components/app/edit-gas-display/index.scss [hunk 6]
```diff
@@ -39,35 +26,10 @@
...
-    background: transparent;
-    color: var(--color-primary-default);
-    font-weight: bold;
-  }
-
   .advanced-gas-controls {
     margin-top: 20px;
   }
 
-  &__education {
```
Comment (georgewrmarshall) [inline ui/components/app/edit-gas-display/index.scss:54]: No className exists for these styles
Screenshot 2023-07-28 at 3 54 21 PM

File: ui/components/app/edit-gas-display/index.scss [hunk 7]
```diff
@@ -39,35 +26,10 @@
...
-
-    button {
-      display: block;
-      margin: 0 auto;
-      background: transparent;
-      color: var(--color-primary-default);
-    }
-  }
-
-  &__warning {
```
Comment (georgewrmarshall) [inline ui/components/app/edit-gas-display/index.scss:65]: Can be replaced with style utility props

**Review by georgewrmarshall — APPROVED** [review]
LGTM! While I was in your branch I removed some of the unused styles. I've updated the PR description with up to date screenshots. Thanks for you contribution @pritam1813

**Review by garrettbear — APPROVED** [review]

**Pull request merged**